docs/snippets/snippets.toml

```diff
@@ -17,35 +17,53 @@
 # Tip: use UTF8 NBSPs (https://unicode-explorer.com/c/00A0) as needed.
 [snippets_ref]
 features = [
-  ["Update rows", [
-    "archetypes/scalar_row_updates",
-    "archetypes/points3d_row_updates",
-    "archetypes/transform3d_row_updates",
-  ]],
-  ["Update columns", [
-    "archetypes/scalar_column_updates",
-    "archetypes/points3d_column_updates",
-    "archetypes/transform3d_column_updates",
-    "archetypes/image_column_updates",  # bonus!
-  ]],
-  ["Partial updates", [
-    "archetypes/points3d_partial_updates",
-    "archetypes/transform3d_partial_updates",
-    "archetypes/mesh3d_partial_updates",
-  ]],
-  ["Send custom data", [
-    "tutorials/any_values",
-    "tutorials/extra_values",
-    "tutorials/custom_data",
-  ]],
-  ["Send columns of custom data", [
-    "howto/any_values_column_updates",
-    "howto/any_batch_value_column_updates",
-  ]],
-  ["Query dataframes", [
-    "reference/dataframe_query",
-    "reference/dataframe_view_query",
-  ]],
+  [
+    "Update rows",
+    [
+      "archetypes/scalar_row_updates",
+      "archetypes/points3d_row_updates",
+      "archetypes/transform3d_row_updates",
+    ],
+  ],
+  [
+    "Update columns",
+    [
+      "archetypes/scalar_column_updates",
+      "archetypes/points3d_column_updates",
+      "archetypes/transform3d_column_updates",
+      "archetypes/image_column_updates",       # bonus!
+    ],
+  ],
+  [
+    "Partial updates",
+    [
+      "archetypes/points3d_partial_updates",
+      "archetypes/transform3d_partial_updates",
+      "archetypes/mesh3d_partial_updates",
+    ],
+  ],
+  [
+    "Send custom data",
+    [
+      "tutorials/any_values",
+      "tutorials/extra_values",
+      "tutorials/custom_data",
+    ],
+  ],
+  [
+    "Send columns of custom data",
+    [
+      "howto/any_values_column_updates",
+      "howto/any_batch_value_column_updates",
+    ],
+  ],
+  [
+    "Query dataframes",
+    [
+      "reference/dataframe_query",
+      "reference/dataframe_view_query",
+    ],
+  ],
 ]
```

Comment:
my format-on-save demanded this 🤷

Reply:
Why isn't this covered by `pixi run toml-fmt`  ?